tests/js/spec/components/acl/featureDisabled.spec.jsx

```diff
@@ -23,6 +23,25 @@ describe('FeatureDisabled', function() {
     expect(wrapper.exists('HelpButton')).toBe(true);
   });
 
+  it('renders with custom message', function() {
+    const customMessage = 'custom message';
+    const wrapper = mount(
+      <FeatureDisabled
+        message={customMessage}
+        feature={'organization:my-feature'}
+        featureName="Some Feature"
+      />,
+      routerContext
+    );
+
+    expect(
+      wrapper
+        .find('Flex')
```

Comment:
Can we use `data-test-id` here? Otherwise it will break tests if this ever changes from `Flex` to a standard div.

Reply:
sure